### DISCUSSION TITLE
Using Prisma with MySQL RDS + Kerberos

### DISCUSSION THREAD

**Discussion by rujhan-arora-astronomer** [prisma/prisma#24689] [Q&A] [answered]
### Question

I have followed this guide - https://www.youtube.com/watch?v=Xb-vezvLvlY , https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/mysql-kerberos.html  to set up a kerberos-enabled MySQL AWS RDS instance.
But when I am using it with Prisma, I am getting this error - Error querying the database: Unknown authentication plugin `dialog'.
<img width="1182" alt="Screenshot 2024-07-02 at 5 32 14 PM" src="https://github.com/prisma/prisma/assets/95576577/a03748b0-264e-4530-8ad9-616f8c51dc64">

Is this a known issue? Or is it a new feature request? Can anyone help me here please?

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment, marked as answer]
Hi @rujhan-arora-astronomer 👋 

Thank you for raising this question.

The error message `Unknown authentication plugin 'dialog'` suggests that Prisma does not currently support the dialog authentication plugin used by your MySQL setup. This issue is similar to other known issues where Prisma does not support certain MySQL authentication plugins. For example, there are open issues for supporting [`mysql_clear_password`](https://github.com/prisma/prisma/issues/10304) and [`sha256_password`](https://github.com/prisma/prisma/issues/14417) plugins.

Given that your issue is related to the `dialog authentication` plugin, it seems to be a new feature request rather than a known issue. You can create a new [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) to request support for the dialog authentication plugin and provide details similar to the ones in the existing issues to help the team understand and prioritize your request.
